### DISCUSSION TITLE
UAV123.json中的NaN怎么处理呀

### DISCUSSION THREAD

**Discussion by XiaoMingDe** [STVIR/pysot#396] [General]
在使用tools/test  测试UAV123时候，跑到第二个视频bike2出现错误       
                                                                       # gt_box = [NaN, NaN, NaN, NaN]
  pred_bbox = list(map(int, pred_bbox))      #  ValueError: cannot convert float NaN to integer

**Comment by ZhiyuanChen** [comment]
抱歉我无法理解你的问题，到底是gt_bbox是NaN还是pred_bbox是NaN？
